Dockerfile_build_ubuntu64

```diff
@@ -1,5 +1,5 @@
 ARG GO_VERSION
```

Comment:
Excited about every change making this smaller. Looks like all the rest of this could be pushed up to the edge repo too - maybe in a follow-up PR?

Reply:
For now, I'll delete all the things from the `apt-get install` that are already covered by the cross-builder.